PubNubSwiftChatSDK/PubNubSwiftChatSDK.docc/Channel.md

```diff
@@ -0,0 +1,78 @@
+# ``PubNubSwiftChatSDK/Channel``
+
+## Topics
+
+### Receiving Updates
+
+- ``streamUpdates(callback:)``
+- ``streamUpdatesOn(channels:callback:)``
+- ``streamReadReceipts(callback:)``
+- ``streamMessageReports(callback:)``
+- ``streamPresence(callback:)``
+
+### Update and Delete a Channel
+
+- ``update(name:custom:description:status:type:completion:)``
+- ``delete(soft:completion:)``
+
+### Typing Indicator
+
+- ``startTyping(completion:)``
+- ``stopTyping(completion:)``
+- ``getTyping(callback:)``
+
+### Presence Management
+
+- ``whoIsPresent(completion:)``
+- ``isPresent(userId:completion:)``
+- ``join(fil:callback:completion:)``
+- ``leave(completion:)``
+- ``streamPresence(callback:)``
```

Comment:
Is this intentional to have streamPresence(callback:) here and in ### Receiving Updates section?

Reply:
Yes, I did it intentionally. However, I can remove it from the `Presence Management` section if you think this could be confusing and leave it in `Receiving Updates` only